pkg/ingress/controller.go

```diff
@@ -166,6 +168,7 @@ func NewController(cfg *config.Config) (*Controller, error) {
 	}
 
 	// recorder
+	utilruntime.Must(apisixScheme.AddToScheme(scheme.Scheme))
```

Comment:
I think to check the return `err` from `apisixscheme.AddToScheme` and return once `err` is not `nil` is better, basically, we don't panic the program if internal errors happen in the same context.

Reply:
The official example I refer to : https://github.com/kubernetes/sample-controller/blob/master/controller.go#L97

Reply:
OK, not a big deal.